src/Microsoft.Framework.Caching.SqlServer/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Data;
+using System.Data.SqlClient;
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    public class Program
+    {
+        private string _connectionString;
+        private string _tableName;
+        private SqlQueries _sqlQueries;
+
+        public void Main(string[] args)
```

Comment:
Extra-nit: consider making this command entry point async and cancellable. Not sure it is needed, i.e. if ctrl+c will just kill the process. Under normal conditions creating the table and the index shouldn't take long but if something blocks it may get annoying.